packages/rps/src/redux/message-service/saga.ts

```diff
@@ -92,10 +92,13 @@ export function* sendMessagesSaga() {
     if (messageState.opponentOutbox) {
       const queue = Queue.GAME_ENGINE;
       const commitment = messageState.opponentOutbox.commitment;
-      const signature = yield signMessage(commitment);
+      // TODO:WALLET_SCRUBBED_OUT
```

Comment:
These comments should make it relatively easy to find the relevant places when we "unscrub" the wallet / replace with RPCs.